src/models.c

```diff
@@ -2807,35 +2807,84 @@ void DrawBillboard(Camera camera, Texture2D texture, Vector3 center, float size,
 
 // Draw a billboard (part of a texture defined by a rectangle)
 void DrawBillboardRec(Camera camera, Texture2D texture, Rectangle source, Vector3 center, float size, Color tint)
+{
+    DrawBillboardPro(camera, texture, source, center, Vector2Zero(), 0.0f, size, tint);
+}
+
+// Draw a billboard (part of a texture defined by a rectangle)
+void DrawBillboardEx(Camera camera, Texture2D texture, Vector3 center, Vector2 rotationPoint, float rotation, float size, Color tint)
```

Comment:
I'd prefer you rename `rotationPoint` as `origin`. It not only determines rotation but also scaling. Naming is also consistent with `DrawTexturePro()`